transformations/character_duplication/transformation.py

```diff
@@ -0,0 +1,53 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
```

Comment:
Please consider adding doc strings, comments and error handling logic.

Reply:
I added a brief doc string in eb09bbcab3d2d47742ef47e353ec315e3ab05627. I believe that the code is simple enough to understand everything without the need of more comments.

Reply:
Please add the description of your arguments, using the doc string convetion: 
`def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...`
as stated in the official doc string convention for Python: https://www.python.org/dev/peps/pep-0257/

Reply:
Don't forget about error handling logic - what happens if the user enters the illegal value for some of the parameters? Will he receive a human-readable message, pointing out what he/she did wrong or a generic Python error log, when the wrong parameter will break the code?